src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Shell/Helpers/ShellListPageHelpers.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.Shell.Commands;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.Shell.Helpers;
+
+public class ShellListPageHelpers
+{
+    private static readonly CompositeFormat WoxPluginCmdCmdHasBeenExecutedTimes = System.Text.CompositeFormat.Parse(Properties.Resources.wox_plugin_cmd_cmd_has_been_executed_times);
+    private readonly SettingsManager _settings;
+
+    public ShellListPageHelpers(SettingsManager settings)
+    {
+        _settings = settings;
+    }
+
+    private ListItem GetCurrentCmd(string cmd)
+    {
+        ListItem result = new ListItem(new ExecuteItem(cmd, _settings))
+        {
+            Title = cmd,
+            Subtitle = Properties.Resources.wox_plugin_cmd_plugin_name + ": " + Properties.Resources.wox_plugin_cmd_execute_through_shell,
+            Icon = new(string.Empty),
```

Comment:
* `\uE768` play
* `\uE756` command prompt, eh, I guess that only makes sense for when the default action is "do it in CMD"

Reply:
I'm going to pitch just 'enter' icon.      Icon = new("\uE751"); // Return Key Icon

Reply:
ok walking that back... I kinda think it will be nice when we have sections ironed out to have history section with the icons and just have a section with NO icon. so I'm going to unresolve this because I'm actually not going to pitch an icon. In fact I'm pitching explicitly not an icon I guess.